packages/roosterjs-editor-dom/lib/list/VListItem.ts

```diff
@@ -216,7 +218,11 @@ export default class VListItem {
      * @param index the list item index
      */
     applyListStyle(rootList: HTMLOListElement | HTMLUListElement, index: number) {
-        const style = getMetadata(rootList) ? getMetadata(rootList) : undefined;
+        const style = getMetadata(rootList)
+            ? (getMetadata(rootList, createNumberDefinition()) as
+                  | NumberingListType
+                  | BulletListType)
+            : undefined;
```

Comment:
Why do like this?

You can just do
const style = getMetadata<NumberingListType|BulletListType>(...);

And if it doesn't have a valid matadata, it will just return null